src/main/java/nl/hsac/fitnesse/fixture/slim/FileFixture.java

```diff
@@ -195,4 +199,11 @@ protected static FunctionalCompletion fileExistsCompletion(String fullname) {
     public boolean pollUntilSizeOfExceeds(String filename, long expectedSize) {
         return repeatUntil(new FunctionalCompletion(() -> exists(filename) && sizeOf(filename) > expectedSize));
     }
+
+    public String appendContainingOnNewline(String filename, String content, boolean onNewLine) {
```

Comment:
I don't think this is the best method name for wiki usage. What do you expect to see in a wiki table row (that's also why extending the sample page will be nice).

I think something like `appendTo(String extraContent, String filename)` would be nicer, and maybe a separate `appendNewLineTo(filename)`, or `appendLineWithTo(content, filename)`. I think a separate method is nicer than always requiring a boolean parameter.